src/theme.js

```diff
@@ -0,0 +1,76 @@
+import PropTypes from "prop-types";
+import { injectGlobal } from "emotion";
+
+// Add font faces:
+injectGlobal`
+  @font-face {
+    font-family: "Lithos";
+    src: url("/static/css/lithos.ttf");
+  }
+  @font-face {
+    font-family: "Bowlby";
+    src: url("/static/css/BowlbyOneSC-Regular.ttf");
+  }
```

Comment:
Note that all fonts are now loaded globally. It seems like the quickest way to deal with flavor-specific fonts for now.